seacrowd/sea_datasets/cebuaner/cebuaner.py

```diff
@@ -0,0 +1,189 @@
+}
+"""
+
+_LOCAL = False
+_LANGUAGES = ["ceb"]
+_DATASETNAME = "cebuaner"
+_DESCRIPTION = """\
+The CebuaNER dataset contains 4000+ news articles that have been tagged by
+native speakers of Cebuano usin gthe BIO encoding schema for the named entity
+recognition (NER) task.
+"""
+
+_HOMEPAGE = "https://github.com/mebzmoren/CebuaNER"
+_LICENSE = Licenses.CC_BY_NC_SA_4_0.value
+_URLS = {
+    "annotator_0": "https://github.com/mebzmoren/CebuaNER/raw/main/data/annotated_data/final-1.txt",
+    "annotator_1": "https://github.com/mebzmoren/CebuaNER/raw/main/data/annotated_data/final-2.txt",
+}
+
+_SUPPORTED_TASKS = [Tasks.NAMED_ENTITY_RECOGNITION]
+_SOURCE_VERSION = "1.0.0"
+_SEACROWD_VERSION = "1.0.0"
+
+
+class CebuaNERDataset(datasets.GeneratorBasedBuilder):
+    """CebuaNER dataset from https://github.com/mebzmoren/CebuaNER"""
+
+    SOURCE_VERSION = datasets.Version(_SOURCE_VERSION)
+    SEACROWD_VERSION = datasets.Version(_SEACROWD_VERSION)
+
+    SEACROWD_SCHEMA_NAME = "seq_label"
+    LABEL_CLASSES = [
+        "O",
+        "B-PER",
+        "I-PER",
+        "B-ORG",
+        "I-ORG",
+        "B-LOC",
+        "I-LOC",
+        "B-OTHER",
+        "I-OTHER",
+    ]
+
+    # There are two annotators in the CebuaNER dataset but there's no canonical
+    # label. Here, we decided to create loaders for both annotators. The
+    # inter-annotator reliability is high so it's possible to treat either as
+    # gold-standard data.
+    dataset_names = sorted([f"{_DATASETNAME}_{annot}" for annot in _URLS.keys()])
+    BUILDER_CONFIGS = []
+    for name in dataset_names:
+        source_config = SEACrowdConfig(
+            name=f"{name}_source",
+            version=SOURCE_VERSION,
+            description=f"{_DATASETNAME} source schema",
+            schema="source",
+            subset_id=name,
+        )
+        BUILDER_CONFIGS.append(source_config)
+        seacrowd_config = SEACrowdConfig(
+            name=f"{name}_seacrowd_{SEACROWD_SCHEMA_NAME}",
+            version=SEACROWD_VERSION,
+            description=f"{_DATASETNAME} SEACrowd schema",
+            schema=f"seacrowd_{SEACROWD_SCHEMA_NAME}",
+            subset_id=name,
+        )
+        BUILDER_CONFIGS.append(seacrowd_config)
+
+    # Create a configuration that loads the annotations of the first annotator
+    # and treat that as the default.
+    BUILDER_CONFIGS.extend([
+        SEACrowdConfig(
+            name=f"{_DATASETNAME}_source",
+            version=SOURCE_VERSION,
+            description=f"{_DATASETNAME} source schema",
+            schema="source",
+            subset_id=_DATASETNAME,
+        ),
+        SEACrowdConfig(
+            name=f"{_DATASETNAME}_seacrowd_{SEACROWD_SCHEMA_NAME}",
+            version=SEACROWD_VERSION,
+            description=f"{_DATASETNAME} SEACrowd schema",
+            schema=f"seacrowd_{SEACROWD_SCHEMA_NAME}",
+            subset_id=_DATASETNAME,
+        ),
+    ])
+
+    DEFAULT_CONFIG_NAME = f"{_DATASETNAME}_1_source"
```

Comment:
this default config name doesn't refer to any built config, haha.
Please change it to `f"{_DATASETNAME}_annotator_1_source"`

Reply:
Oh I just realized, maybe this should be the base config `cebuaner_source` (which also loads annotator 1)?

Reply:
yeah that's okay too